omc3/optics_measurements/crdt.py

```diff
@@ -47,23 +49,34 @@
 ]
 
 
-def calculate(measure_input, input_files, invariants, header):
-
+def calculate(measure_input: DotDict, input_files: InputFiles, invariants, header):
+    """ Calculate the CRDT values. 
+    
+    Todo: https://github.com/pylhc/omc3/issues/456 
+    """
     LOGGER.info("Start of CRDT analysis")
+    
+    # Todo: only on-momentum required? If not, remove this or set `dpp_value=None`, see #456
+    dpp_value = 0  
+    invariants = {plane: inv[input_files.dpp_frames_indices(plane, dpp_value)] for plane, inv in invariants.items()}
```

Comment:
This is not so explicit what this line does. Maybe a comment? It's keeping only the right rows?